cg/services/analysis_starter/configurator/file_creators/gene_panel.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+
+from cg.apps.scout.scoutapi import ScoutAPI
+from cg.constants import FileExtensions, GenePanelMasterList
+from cg.constants.gene_panel import GenePanelCombo, GenePanelGenomeBuild
+from cg.io.txt import write_txt
+from cg.services.analysis_starter.configurator.file_creators.utils import (
+    get_case_id_from_path,
+    get_genome_build,
+)
+from cg.store.models import Case
+from cg.store.store import Store
+
+
+class GenePanelFileCreator:
+    def __init__(self, store: Store, scout_api: ScoutAPI):
+        self.store = store
+        self.scout_api = scout_api
+
+    @staticmethod
+    def get_file_path(case_path: Path) -> Path:
+        return Path(case_path, "gene_panels").with_suffix(FileExtensions.BED)
+
+    def create(self, case_path: Path) -> None:
+        file_path: Path = self.get_file_path(case_path=case_path)
```

Comment:
This static method is not used anywhere else, we could remove it
```suggestion
    def create(self, case_path: Path) -> None:
        file_path= Path(case_path, "gene_panels").with_suffix(FileExtensions.BED)
```

Reply:
I consider this flavour. I like having the path separate so I will keep it for now.